content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
-- [Versioned Documentation](#versioned-documentation)
-- [Something Missing?](#something-missing)
+- [Փորձել React-ը](#try-react)
+- [Սովորել React](#learn-react)
+- [Մնալ տեղեկացված](#staying-informed)
+- [Տարբերակված փաստաթղթավորում](#versioned-documentation)
+- [Ինչ-որ բա՞ն է բաց թողնված](#something-missing)
 
-## Try React {#try-react}
+## Փորձել React-ը {#try-react}
 
-React has been designed from the start for gradual adoption, and **you can use as little or as much React as you need.** Whether you want to get a taste of React, add some interactivity to a simple HTML page, or start a complex React-powered app, the links in this section will help you get started.
+React-ը իսկզբանե նախագծված է եղել այնպես, որ հնարավոր լինի աստիճանաբար ինտեգրել, և **դուք կարող եք օգտագործել այնքան քիչ կամ այնքան շատ React, որրքան դուք ցանկանում եք։** Անկախ այն բանից, ցանկանում եք «համտեսել» React-ը, ավելացնել մի քիչ ինտերակտիվություն հասարակ HTML էջում, թե սկսել բարդ React հավելված\` այս բաժնի հղումները կօգնեն ձեզ անել առաջին քայլերը։
 
-### Online Playgrounds {#online-playgrounds}
+### Ինտերակտիվ օնլայն խմբագրիչներ {#online-playgrounds}
 
-If you're interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world), [CodeSandbox](https://codesandbox.io/s/new), or [Glitch](https://glitch.com/edit/#!/remix/starter-react-template).
+Եթե դուք ցանկանում եք խաղալ React-ի հետ, դուք կարող եք օգտագործել ինտերակտիվ օնլայն խմբագրիչներ։ Փորձիր «Ողջույն, աշխարհ» կաղապարը [CodePen-ում](codepen://hello-world), [CodeSandbox-ում](https://codesandbox.io/s/new) կամ [Glitch-ում](https://glitch.com/edit/#!/remix/starter-react-template)։
 
-If you prefer to use your own text editor, you can also [download this HTML file](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so we'd only recommend using this for simple demos.
+Եթե դուք նախընտրում եք օգտագործել ձեր սեփական տեքստային խմբագրիչը, դուք կարող եք նաև [ներբեռնել այս HTML ֆայլը](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), փոփոխել այն և բացել լոկալ ֆայլային համակարգից ձեր զննարկչի մեջ։ Այն, runtime, կոդի փոխակերպում է իրականացնում, որը դանդաղ է, այդ իսկ պատճառով մենք կառաջարկենք այն օգտագործել միայն հասարակ ցուցադրությունների համար։
 
-### Add React to a Website {#add-react-to-a-website}
+### Ավելացնել React-ը կայքում {#add-react-to-a-website}
 
-You can [add React to an HTML page in one minute](/docs/add-react-to-a-website.html). You can then either gradually expand its presence, or keep it contained to a few dynamic widgets.
+Դուք կարող եք [մեկ րոպեում ավելացնել React-ը HTML էջում](/docs/add-react-to-a-website.html)։ Հաջորդիվ, դուք կարող եք կամ աստիճանաբար ընդլայնել նրա ներկայությունը կամ պահել այն մի քանի դինամիկ widget-ների համար։
```

Comment:
"contained to"-ն պարունակվելը չի՞, մի քիչ չհասկացա իմաստը

Reply:
մեկ էլ "presence"֊ը իհարկե ներկայություն, առկայությունն ա, բայց "ընդլայնել  ներկայությունը" ո՞րն ա, կարողա՞ նկատի ունի արտաքին տեսքը ընդլայնել։